frontend/components/Notebook.js

```diff
@@ -9,21 +9,17 @@ let CellMemo = ({
     cell_input,
     cell_input_local,
     notebook_id,
-    on_update_doc_query,
-    on_cell_input,
     cell_dependencies,
-    on_focus_neighbor,
     selected,
-    selected_cells,
     focus_after_creation,
     force_hide_input,
     is_process_ready,
     disable_input,
     show_logs,
     set_show_logs,
     nbpkg,
+    variables_in_all_notebook,
 }) => {
-    const selected_cells_diffable_primitive = (selected_cells || []).join("")
```

Comment:
this is one of the places we diffed selected_cells. we don't need to thanks to the new actions!